README.md

```diff
@@ -27,63 +27,54 @@ We also have edge agents for other platforms:
 - Agent: A component using all other building blocks, provides basic edge agent capabilities, including implementing DIDComm V2 protocols.
 
 ## Getting started
+We highly recommend you [check out the online docs](https://hyperledger.github.io/identus-docs/docs/getting-started) :world_map:
```

Comment:
this is the MUST HAVE line for this PR

Reply:
Agree, closing conversation so we  can merge soon